metricbeat/module/nats/varz/_meta/fields.yml

```diff
@@ -0,0 +1,77 @@
+- name: varz
+  type: group
+  description: >
+    Contains nats var related metrics
+  fields:
+    - name: server_id
+      type: keyword
+      description: >
+        The server ID
+    - name: now
+      type: date
+      description: >
+        Server time of metric creation
+    - name: uptime
+      type: long
+      description: >
+        The period the server is up
+    - name: mem
+      type: integer
+      description: >
+        The current memory usage of NATS process
+    - name: cores
+      type: integer
```

Comment:
This is unexpected. Is it possible that it returns an integer value? I'd bet this value is a double because I'm expecting it to return something like _1.5 cores_ but maybe I'm wrong.

Reply:
You are right about the cpu. It should be double but the cores is integer. This is how nats
endpoint returns the [values.](https://github.com/nats-io/gnatsd/blob/master/server/monitor.go#L843-L842)

Reply:
Awesome, just wanted to be sure that it wasn't a leftover.